projects/microphysics/train/limited.yaml

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
i downloaded this from wandb since it wasn't in master yet and is an active target for development